conf/stopo.conf.part

```diff
@@ -946,6 +946,28 @@ source src_ch_swisstopo_geologie_reflexionsseismik : def_searchable_features
         from geol.view_geologie_reflexionsseismik_sphinx
 }
 
+source src_ch_swisstopo_geologie_hartsteinabbau : def_searchable_features
+{
+    sql_db = stopo_${DBSTAGING}
+    sql_query = \
+        WITH data as ( \
+            SELECT bgdi_id, obname, cpkind, the_geom from geol.hartstein_production UNION ALL \
+            SELECT bgdi_id, obname, cpkind, the_geom from geol.hartstein_mining \
+        ) \
+        SELECT bgdi_id as id \
```

Comment:
is bgdi_id going to be unique?

Reply:
yes it has been set up here:
https://github.com/geoadmin/bgdi-scripts/pull/1643/commits/13e765e1645830478f231f849b95fbff0b189d41